samples/wifi/scan/src/main.c

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2018 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
I know this was based on shell sample, but this should really be 2022 + Nordic-5-clause license.